avocado/core/test.py

```diff
@@ -783,3 +783,17 @@ class ReplaySkipTest(SkipTest):
     """
 
     _skip_reason = "Test skipped due to a job replay filter!"
+
+
+class TestLoaderError(Test):
```

Comment:
It's not necessarily just `TestLoaderError`, it's just a generic `TestError` test.